### PR TITLE
570 add support for setting coordinates for location tags

### PR DESCRIPTION
**Pull request by LinoH5**
- Allow Curators to set coordinates for each location tag, via a map in the `LocationManagementDialog`

### CODE REVIEW

File: projects/bp-gallery/src/components/views/location-curating/LocationEntry.scss [hunk 1]
```diff
@@ -71,7 +71,7 @@
         .location-synonyms {
           display: flex;
           flex-direction: row;
-          overflow: scroll;
+          overflow: auto;
```
Comment (ole1711) [inline projects/bp-gallery/src/components/views/location-curating/LocationEntry.scss:74]: wo tailwind

**Review by ole1711 — APPROVED** [review]

File: projects/bp-gallery/src/components/views/location-curating/LocationEntryActions.tsx [hunk 2]
```diff
@@ -37,6 +39,9 @@ const LocationEntryActions = ({
 
   return (
     <div className='location-action-buttons-container'>
+      {locationTag.coordinates && (
+        <IconButton>{<LocationOn className='text-[#446ab7]' />}</IconButton>
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/location-curating/LocationEntryActions.tsx:43]: remove `{}`

File: projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.tsx [hunk 3]
```diff
@@ -35,6 +44,32 @@ import {
...
+      'iconAnchor',
+      'popupAnchor',
+      'shadowSize',
+      'shadowAnchor',
+    ]),
+    iconUrl: myMarkerIcon,
+    shadowUrl: markerShadow,
+  });
+
+  return position ? <Marker icon={myIcon} position={position} /> : <div></div>;
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.tsx:70]: maybe replace the `div` with `null`

File: projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.tsx [hunk 4]
```diff
@@ -108,6 +143,45 @@ const LocationManagementDialogPreset = ({
...
+
+  const map = useRef<Map>(null);
+  const [position, setPosition] = useState<LatLng | undefined>(
+    locationTag.coordinates
+      ? new LatLng(locationTag.coordinates.latitude, locationTag.coordinates.longitude)
+      : undefined
+  );
+  const initialMapValues = useMemo(() => {
+    return {
+      center: locationTag.coordinates ? position : new LatLng(51.8392573, 10.5279953),
```
Comment (MariusDoe) [inline projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.tsx:158]: Maybe put this in a named constant?

**Review by MariusDoe — CHANGES_REQUESTED** [review]

**Review by MariusDoe — APPROVED** [review]

**Pull request merged**